src/components/sbb-panel/styles/sbb-panel.component.scss

```diff
@@ -4,52 +4,44 @@
 @include host-component-properties;
 
 :host {
-  display: block;
+  --panel-opacity: 0.9;
+  --panel-padding-inline: var(--sbb-spacing-responsive-m-small);
+  --panel-padding-block: var(--sbb-spacing-responsive-s-small);
+  --background-color: var(--sbb-color-red-default);
+  --text-margin-block-end: var(--sbb-spacing-fixed-6x);
 
   // TODO: missing design tokens
-  --font-family: 'SBBWeb Light';
-  --letter-spacing: 0.01em;
-  --line-height: 1.3;
-  --font-size: #{px-to-rem-build(24)};
   --panel-width: #{px-to-rem-build(223)};
   --panel-height: #{px-to-rem-build(167)};
-  --button-margin: var(--sbb-spacing-responsive-xs-small);
-  --panel-padding-inline: var(--sbb-spacing-responsive-m-small);
-  --panel-padding-block: var(--sbb-spacing-responsive-s-small);
   --triangle-height: #{px-to-rem-build(33)};
```

Comment:
<img width="426" alt="Bildschirmfoto 2022-06-30 um 15 55 31" src="https://user-images.githubusercontent.com/29278232/176696948-777da46f-8d6f-4e49-8110-b360cac7f613.png">
Sometimes there is a line visible between the triangle and the content box. To avoid that I have the idea to not use separate elements (before / after) but to use only one element which has a `clip-path: polygon(0 0, 100% 45px, 100% calc(100% - 45px), 0px 100%);` on it. 45px has to be replaced with var(--triangle-height). moreover the padding of the box has to be calculated like sbb-spacing-responsive-s + triangle-height.